Tests/RevenueCatUITests/CustomerCenter/CustomerCenterViewModelTests.swift

```diff
@@ -1336,7 +1385,7 @@ private extension CustomerCenterViewModelTests {
         """)
     }
 
-    static let customerInfoWithoutSubscriptions: CustomerInfo = {
+    static let customerInfoWithExpiredSubscriptions: CustomerInfo = {
```

Comment:
I feel we should have both, `customerInfoWithoutSubscriptions` and `customerInfoWithExpiredSubscriptions` to test all cases correctly